firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallationException.java

```diff
@@ -12,43 +12,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.firebase.installations.remote;
+package com.google.firebase.installations;
 
 import androidx.annotation.NonNull;
 import com.google.firebase.FirebaseException;
 
-/** The class for all Exceptions thrown by {@link FirebaseInstallationServiceClient}. */
-public class FirebaseInstallationServiceException extends FirebaseException {
+/** The class for all Exceptions thrown by {@link FirebaseInstallations}. */
+public class FirebaseInstallationException extends FirebaseException {
 
   public enum Code {
```

Comment:
Code is usually used for network response, how about Status?

Reply:
BTW, I think you probably should keep two kinds of exceptions, one is used internally for the Service client, one is used to expose to the caller app.

Reply:
Done. Thanks.